app/src/main/java/org/mifos/mobilebanking/api/local/PreferencesHelper.java

```diff
@@ -161,4 +176,13 @@ public void setSentTokenToServer(boolean sentTokenToServer) {
     public boolean sentTokenToServerState() {
         return getBoolean(SENT_TOKEN_TO_SERVER, false);
     }
+
+    public void setBaseUrl(String baseUrl) {
+        putString(context.getString(R.string.pref_base_url_key), baseUrl);
+    }
+
+    public String getBaseUrl() {
+        return getString(context.getString(R.string.pref_base_url_key),
```

Comment:
Don't use key value name from strings.xml, please define key value on top of class like these https://github.com/openMF/mifos-mobile/pull/716/files#diff-4afc21547670976db853693ae9f5d17cR29